src/overlays/actors/ovl_En_Zl2/z_en_zl2.c

```diff
@@ -27,265 +101,1664 @@ const ActorInit En_Zl2_InitVars = {
+        Matrix_Translate(-1019.0f, -26.0f, 0.0f, MTXMODE_APPLY);
+        Matrix_ToMtx(sp74 + 2, "../z_en_zl2.c", 1120);
+        Matrix_Pull();
+        Matrix_Push();
+        Matrix_Translate(467.0f, 265.0f, 389.0f, MTXMODE_APPLY);
+        Matrix_Get(&sp34);
+        func_800D20CC(&sp34, &sp2C, 0);
+        if (func_800C0D28(globalCtx) == 0) {
+            func_80B4EE38(this, sp2C.y, 9);
+            func_80B4F230(this, sp2C.x, 10);
+            func_80B4EF64(this, sp2C.z, 11);
+        }
+        Matrix_RotateRPY(unk_1DC[9] + kREG(40), unk_1DC[10] + kREG(41), unk_1DC[11] + kREG(42), MTXMODE_APPLY);
+        Matrix_Translate(-427.0f, -1.0f, -3.0f, MTXMODE_APPLY);
+        Matrix_ToMtx(sp74 + 3, "../z_en_zl2.c", 1145);
+        Matrix_Get(&sp34);
+        func_800D20CC(&sp34, &sp2C, 0);
+        if (func_800C0D28(globalCtx) == 0) {
+            func_80B4EE38(this, sp2C.y, 12);
+            func_80B4F230(this, sp2C.x, 13);
+            func_80B4EF64(this, sp2C.z, 14);
+        }
+        Matrix_RotateRPY(unk_1DC[12] + kREG(43), unk_1DC[13] + kREG(44), unk_1DC[14] + kREG(45), MTXMODE_APPLY);
+        Matrix_Translate(-446.0f, -52.0f, 84.0f, MTXMODE_APPLY);
+        Matrix_ToMtx(sp74 + 4, "../z_en_zl2.c", 1164);
+        Matrix_Pull();
+        Matrix_Push();
+        Matrix_Translate(467.0f, 265.0f, -389.0f, MTXMODE_APPLY);
+        Matrix_Get(&sp34);
+        func_800D20CC(&sp34, &sp2C, 0);
+        if (func_800C0D28(globalCtx) == 0) {
+            func_80B4EE38(this, sp2C.y, 15);
+            func_80B4F230(this, sp2C.x, 16);
+            func_80B4EF64(this, sp2C.z, 17);
+        }
+        Matrix_RotateRPY(unk_1DC[15] + kREG(46), unk_1DC[16] + kREG(47), unk_1DC[17] + kREG(48), MTXMODE_APPLY);
+        Matrix_Translate(-427.0f, -1.0f, 3.0f, MTXMODE_APPLY);
+        Matrix_ToMtx(sp74 + 5, "../z_en_zl2.c", 1189);
+        Matrix_Get(&sp34);
+        func_800D20CC(&sp34, &sp2C, 0);
+        if (func_800C0D28(globalCtx) == 0) {
+            func_80B4EE38(this, sp2C.y, 18);
+            func_80B4F230(this, sp2C.x, 19);
+            func_80B4EF64(this, sp2C.z, 20);
+        }
+        Matrix_RotateRPY(unk_1DC[18] + kREG(49), unk_1DC[19] + kREG(50), unk_1DC[20] + kREG(51), MTXMODE_APPLY);
+        Matrix_Translate(-446.0f, -52.0f, -84.0f, MTXMODE_APPLY);
+        Matrix_ToMtx(sp74 + 6, "../z_en_zl2.c", 1208);
+        Matrix_Pull();
+        Matrix_Pull();
+        this->unk_24C = 1;
+    }
+    return 0;
+}
+
+void EnZl2_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx, Gfx** gfx);
+#ifdef NON_MATCHING
+// Stack issue in two instructions - made much better with the pad
+void EnZl2_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx, Gfx** gfx) {
+    s32 pad[2];
+    EnZl2* this = THIS;
+    Player* player;
+
+    if (limbIndex == 10) {
+        if ((this->unk_254 != 0) && (globalCtx->csCtx.frames >= 0x384)) {
```

Comment:
Frames should all be in decimal imo. (Relevant throughout the file)
```suggestion
        if ((this->unk_254 != 0) && (globalCtx->csCtx.frames >= 900)) {
```